bin/plugin/commands/common.js

```diff
@@ -41,44 +40,16 @@ exports.getModuleData = async ( modulesDir ) => {
 
 	return moduleFiles
 		.map( ( moduleFile ) => {
+			const moduleFileContent = fs.readFileSync( moduleFile, 'utf8' );
```

Comment:
Another note that we should always catch errors for synchronous functions, especially I/O ones. I'd suggest wrapping this in a `try catch`.